### PR TITLE
added 'no-verify' flag to ignore ssl cert verification

### PR DESCRIPTION
**Pull request by jarrettchisholm**
We have custom certs, so this allows us to use the tool.

### CODE REVIEW

File: jiracli/__init__.py [hunk 1]
```diff
@@ -408,6 +415,11 @@ def main():
     args = parse_args()
     setup_logging(log, args['debug'])
     conf = config_get(user_config_path)
+
+    # Override config setting if user requested to ignore ssl cert verification
+    if args['no_verify']:
+        conf['verify'] = '0'
```
Comment (toabctl) [inline jiracli/__init__.py]: please set it to the boolean type ```False```

File: jiracli/__init__.py [hunk 2]
```diff
@@ -124,8 +124,13 @@ def config_get(config_path):
 
 
 def jira_obj_get(conf):
+    verify = True
+    if 'verify' in conf and conf['verify'] in ['0', 'false', 'False', 'no']:
```
Comment (toabctl) [inline jiracli/__init__.py]: Can you use https://docs.python.org/2/library/configparser.html#ConfigParser.RawConfigParser.getboolean ?

**Comment by toabctl** [comment]
Thanks for the patch. Some comments inline and please squash both commits.

**Comment by jarrettchisholm** [comment]
Done

File: jiracli/__init__.py (hunk 2 above)
Comment (jarrettchisholm) [inline jiracli/__init__.py]: Not here - it is within a `dict` at this point.  I could rejig the way config is loaded - maybe getboolean as a special case for this config key, or maybe iterate over all values and convert to boolean dynamically (i.e. if value is 'False', 'false', 'no' or '0' make it False, etc)...thoughts?

**Comment by toabctl** [comment]
travis failed now. pep8 errors

**Comment by toabctl** [comment]
@jarrettchisholm friendly reminder about fixing the pep8 error. You can run the test manually with ```tox -epep8```

**Comment by mpenning** [comment]
@toabctl, this is a pretty important feature to have... if the PR isn't going to happen, could you rewrite as you like?

**Comment by toabctl** [comment]
Adjusted the commit a bit and merged it manually. Thanks!

**Pull request closed without merging**